### ISSUE TITLE
First Aid Kits Disassemble incorrectly (sometimes)

### ISSUE THREAD

**Issue by Xpyder** [CleverRaven/Cataclysm-DDA#12100]
a stack of first aid kits seems to disassemble to exactly the same materials as a pair of kits would.

For example, you would expect 10 kits to disassemble into (possibly) 100 tapes but it only ever creates 20.

**Comment by BevapDin** [comment]
> For example, you would expect 10 kits to disassemble into (possibly) 100 tapes but it only ever creates 20.

They actually do. Just go ahead and try it out. The default stack size is 2, each dissembling action consumes as much items as you would get when you had crafted it and gives you back as much items as you would have used when crafting it.

Disassembling a stack of 10 first aid kits gives you back 20 tape, 8 aspirin, 2 bandage and 1 disinfectant - but it also consumes only 2 first aid kits, you still have 8 further kits to disassemble. In the end you get what you would have used to make it.

**Comment by Xpyder** [comment]
I did just try it in my build (bcdbefd) from 4/13. 
disassembled 7 together gave 20 medical tapes
then 3 sets of 2 gave 20 medical tapes each (60 total)

**Comment by Xpyder** [comment]
I tested it fresh this morning and it worked as you said.  I also know that when I tested it before it ate all my kits (otherwise I wouldn't have noticed it was supposed to be eating 2 at a time).

I've also had it not allow me to fill tanks on vehicles after using it for a while (it says there's no part there or the part is damaged).  I think there's some kind of underlying issue with activating certain events that occurs after some as yet unknown trigger condition.  It might be related to the Advanced Inventory Bugs, or just running for too many rounds such that the loaded memory fills up and starts getting pushed around (I run for 4-6 hours at a time)
